lib/install/validate/fix_container.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
```

Comment:
I see no unit tests for this file. Please, talk to me in person before you write one. I can show you some good practices.